app/Http/Controllers/TaskController.php

```diff
@@ -12,23 +16,34 @@ class TaskController extends Controller
      */
     public function index()
     {
-        return Task::all();
+        return view('tasks.index', [
+            'tasks' => DB::table('tasks')->get(),
+        ]);
     }
 
     /**
      * Show the form for creating a new resource.
      */
     public function create()
     {
-        //
+        return view('tasks.create');
     }
 
     /**
      * Store a newly created resource in storage.
      */
-    public function store(Request $request)
+    public function store(ValidateTaskRequest $request)
     {
-        //
+        $validated = $request->validated();
+
+        DB::table('tasks')->insert([
+            'content' => $validated['content'],
+            'name' => $validated['name'],
+            'username' => Auth::user()->username,
+            'user_id' => Auth::user()->id,
+        ]);
+
+        return $this->index();
```

Comment:
```suggestion
        return redirect()->back(); // return redirect()->route(...);
```
dùng cách này nha... ko nên gọi qua lại như thế này

Reply:
Em đã sửa ạ